### DISCUSSION TITLE
Pet Images

### DISCUSSION THREAD

**Discussion by e-laflamme** [CSC207-2022F-UofT/course-project-my-dog-ate-my-hw#70] [Show and tell]
So I made a few pets for us finally!! Here they are (the last one is my fav):
![corgi](https://user-images.githubusercontent.com/113330224/205869735-c45a1fa8-9422-4208-8c04-fd269e0a7455.PNG)
![beagle](https://user-images.githubusercontent.com/113330224/205869774-6c3dec3a-a7ec-4aaa-b1b7-0493cd6563f6.PNG)
![husky](https://user-images.githubusercontent.com/113330224/205869788-1660da40-26ef-463b-8e0e-9b0e22a92997.PNG)
![pomeranian](https://user-images.githubusercontent.com/113330224/205869800-20301382-fea6-4439-b3fa-9aca7779bdcd.PNG)

I made a background that I put them in front of, do you guys like it? Or should we just have a solid color background?

**Comment by XinyingXu** [comment]
These pets look amazing! And I think the background looks good.
